pkg/scheduler/core/scheduling_queue.go

```diff
@@ -150,14 +156,21 @@ type UnschedulablePods interface {
 // pods that are already tried and are determined to be unschedulable. The latter
 // is called unschedulableQ.
 type PriorityQueue struct {
-	lock sync.RWMutex
-	cond sync.Cond
+	lock  sync.RWMutex
+	cond  sync.Cond
+	stop  <-chan struct{}
+	clock util.Clock
 
 	// activeQ is heap structure that scheduler actively looks at to find pods to
 	// schedule. Head of heap is the highest priority pod.
-	activeQ *Heap
+	activeQ *util.Heap
 	// unschedulableQ holds pods that have been tried and determined unschedulable.
 	unschedulableQ *UnschedulablePodsMap
+	// podBackoff holds pods which are awaiting a backoff period before being added to activeQ
```

Comment:
I think the comment would be clearer if you change "holds" to "tracks". Pods awaiting a backoff period will still be hold in the unschedulableQ. The current comment may make readers think that they are moved from unschedulableQ to podBackoff while waiting.